### PR TITLE
first step in #32

### PR DESCRIPTION
**Pull request by jread-usgs**
2 hrs. makes this:
circle_map_total

### CODE REVIEW

File: scripts/visualize/make_circle_map.R [hunk 1]
```diff
@@ -33,6 +33,76 @@ visualize.make_circle_map <- function(viz){
...
+  
+  for (j in seq_len(length(circle_sp_transf))){
+    # this is in a loop because it is throwaway code: 
+    r <- circle_sp_transf$radius_total[j] *rad_multiplier
+    c.x <- coordinates(circle_sp_transf)[j, ][['x']]
+    c.y <- coordinates(circle_sp_transf)[j, ][['y']]
+    for (cat in categories){
+      cat_angle <- circle_sp_transf[[cat]][j] / circle_sp_transf[['total']][j]*2*pi
+      if (cat == head(categories, 1L)){
+        # start the first category mirrorer relative to the top
```
Comment (aappling-usgs) [inline scripts/visualize/make_circle_map.R]: mirrorer?

File: scripts/visualize/make_circle_map.R [hunk 2]
```diff
@@ -33,6 +33,76 @@ visualize.make_circle_map <- function(viz){
...
+  
+  
+  png(viz[["location"]], width = 10, height = 6.0, res=450, units = 'in')
+  
+  par(mai=c(0,0,0,0), omi=c(0,0,0,0), xaxs = 'i', yaxs = 'i')
+  plot(sf::st_geometry(map_data_usa), col = "#f1f1f1", border = "white")
+  plot(sf::st_geometry(map_data_county), col = NA, border = "white", add=TRUE, lwd = 0.5)
+  
+  # start w/ irrigation, fill around the categories (make sure they add up!)
+  # note: they don't add up because we aren't including all
```
Comment (aappling-usgs) [inline scripts/visualize/make_circle_map.R]: my guess is we'll want an "Other" category if we decide to include this in the final viz

File: scripts/visualize/make_circle_map.R (hunk 1 above)
Comment (jread-usgs) [reply scripts/visualize/make_circle_map.R]: geez, terrible comment by me. What I meant was the first category (in this case) will have a slice centered on the 90° like this:
image

File: scripts/visualize/make_circle_map.R (hunk 2 above)
Comment (jread-usgs) [reply scripts/visualize/make_circle_map.R]: agreed

File: scripts/visualize/make_circle_map.R (hunk 1 above)
Comment (aappling-usgs) [reply scripts/visualize/make_circle_map.R]: oh, now i see where the mirror concept comes in :+1:

**Pull request merged**